integr8ly/integreatly-operator
# MGDAPI-4800 make 3cale operator cluster scoped

# Issue link

[MGDAPI-4800](https://issues.redhat.com/browse/MGDAPI-4800)

# What

Make the 3scale operator cluster scoped


# Verification steps
Install locally
```bash
INSTALLATION_TYPE=managed-api make cluster/prepare/local
INSTALLATION_TYPE=managed-api USE_CLUSTER_STORAGE=true make deploy/integreatly-rhmi-cr.yml
INSTALLATION_TYPE=managed-api make code/run
```
In installed operators confirm that the 3scale operator is watching all namespaces
![image](https://user-images.githubusercontent.com/16667688/199529832-d8bede08-ce2a-4271-8606-6aa52ca484aa.png)

Verify upgrade
